src/app/modules/main/chat_section/controller.nim

```diff
@@ -230,4 +230,34 @@ method rejectContactRequest*(self: Controller, publicKey: string) =
   self.contactService.rejectContactRequest(publicKey)
 
 method blockContact*(self: Controller, publicKey: string) =
-  self.contactService.blockContact(publicKey)
\ No newline at end of file
+  self.contactService.blockContact(publicKey)
+
+proc convertPubKeysToJson(self: Controller, pubKeys: string): seq[string] =
+  return map(parseJson(pubKeys).getElems(), proc(x:JsonNode):string = x.getStr)
+
+method addGroupMembers*(self: Controller, chatId: string, pubKeys: string) =
+  self.chatService.addGroupMembers(chatId, self.convertPubKeysToJson(pubKeys))
+  
+method removeMemberFromGroupChat*(self: Controller, chatId: string, pubKey: string) =
+   self.chatService.removeMemberFromGroupChat(chatId, pubKey)
+
+method renameGroupChat*(self: Controller, chatId: string, newName: string) =
+  self.chatService.renameGroupChat(chatId, newName)
+
+method makeAdmin*(self: Controller, chatId: string, pubKey: string) =
```

Comment:
Small improvement, but all the `chatId`s above can probably be removed since the controller already knows n which chat it is (it should be `activeItemId` I think, but it might depend on if it is a community or something, @saledjenic can probably confirm)

Reply:
@jrainville @borismelnik actually this is a `chat section module` the only thing which we're sure here is actually section id (what is in case of community section community id), so keeping `chatId` for methods here is ok, since we're able to do some actions for chat/channel which is not an active one.

just to make this things clear, so structure is:
```
|->main module
|---> chat section module
|------> chat content module
|---------> messages module
|---------> users module
|---------> input area module
```
That means if we're calling some service proc:
- from the `controller` of chat section module, we're sure only about `sectionId` (which is in case of community actually `communityId`)
- from the `controller` of chat content module, we're sure about `sectionId` (which is in case of community actually `communityId`) and about `chatId` which is chat/channel depends in which section we operate